helm/polaris/values.yaml

```diff
@@ -0,0 +1,370 @@
+# -- The number of replicas to deploy (horizontal scaling).
+# Beware that replicas are stateless; don't set this number > 1 when using in-memory meta store manager.
+replicaCount: 1
+
+initImage:
```

Comment:
nit: I'd call it "toolsImage"... "init" sounds too strong, it feels like this image already contains init logic (which it does not).

Reply:
I think init here mainly refers to init container as that is generic in k8s. But yes, we can call it toolsImage.